### DISCUSSION TITLE
Implement Group Looting from MoP

### DISCUSSION THREAD

**Discussion by NinjaWitaSniper** [chromiecraft/chromiecraft#3563] [Suggestions]
In Mists of Pandaria when you kill multiple mobs, you only have to loot one body instead of looting multiple bodies.

How hard would it be to implement this QoL feature? Is it even possible?

**Comment by acidmanifesto** [comment]
> In Mists of Pandaria when you kill multiple mobs, you only have to loot one body instead of looting multiple bodies.
> 
> How hard would it be to implement this QoL feature? Is it even possible?

This is as best as i could get it over 2 years agos before refactoring it for azerothcore
https://github.com/azerothcore/azerothcore-wotlk/compare/master...acidmanifesto:Azerothcore-AOE-Branch

The big issue is the games world is not balanced for aoe looting.

the client has no distinguishing way of determine multiple people looting same area of range corpses.

Client lacks the ability to stack and add it all in the loot window which then leads to a auto loot of all things in said radius.

We then added a auto mail feature to mail the items to the player if they do not have the space.

Will only work if not in group.

335 client lacks the group looting op code that is present in mop.

Not blizzlike in the realm of 335. 

**So to answer your questions:**
How hard would it be to implement this QoL feature? 
Not hard and already is available opensource.

Is it even possible?
It is possible but not in the way as MOP handles its
